src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1132,15 +1132,10 @@ public OctoTentacle getTentacle()
      */
     void handleIncomingRtcp(PacketInfo packetInfo, AbstractEndpoint source)
     {
+        String sourceEpId = source != null ? source.getID() : null;
```

Comment:
> Is there anything different for RTCP at this level? Can we merge the logic for handleIncomingRtp and handleIncomingRtcp (perhaps rename to handleIncomingPacket)? This would simplify the code and avoid the unnecessary clone().

i'm fine with that as long as it doesn't get vetoed /cc @bbaldino

Reply:
See my other comment, it applies here as well

Reply:
this is now taken care of